topcoder-platform/freeCodeCamp
# PROD-2031 FCC Using TC Auth0 on FQDN -> PROD-2031_auth0

https://topcoder.atlassian.net/browse/PROD-2196

To verify this ticket:

1. copy the sample.topcoder.env file to .env
2. add the auth0 Account-App-New-Flow client ID and secret
- https://manage.auth0.com/dashboard/us/topcoder-dev/applications/BXWXUWnilVUPdN01t2Se29Tw2ZYNGZvH/settings
3. > npm run develop
4. > npm run start-proxy
5. > npm run start-proxy-api
6. go to the standalone FCC app at https://fcc.topcoder-dev.com:4431
7. click the Sign In button
- you might need to run FCC a private window to be permitted to access the api using the ssl proxy
8. log in using your topcoder-dev credentials
9. complete a step in a module

Verify in the Mongo DB that your progress appears as the user w/the email attached to your topcoder-dev creds
